src/components/Global/LinkAccountComponent/index.tsx

```
@@ -289,66 +300,59 @@ export const GlobaLinkAccountComponent = ({ accountNumber, onCompleted }: IGloba
+                    return
+                }
 
-            console.log('Fetching updated user data')
-            await fetchUser()
+                const bridgeAccountId = createExternalAccountRes.data.id
 
-            onCompleted ? onCompleted() : setCompletedLinking(true)
-        } catch (error) {
-            console.error('Error in handleSubmitLinkIban:', error)
-            let errorMessage = 'Failed to link bank account'
+                if (!!bridgeAccountId) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)